en/SUMMARY.md

```diff
@@ -1,96 +1,113 @@
 
 ## What is Cocos2d-x
-
 - [About Cocos2d-x](./about.md)
 - [Why choose Cocos2d-x]()
-- [...]()
+- [How to learn]()
+- [Where to get help]()
+- [How to contribute]()
 
 ## Basic Cocos2d-x Concepts
-
 - [Main Components](./basic_concepts.md)
 - [Director]()
-- [Scene]()
+- [Scenes and the Scene graph]()
 - [Sprites]()
-- [...]()
+- [Actions]()
 
 ## Sprites
-
 - [What are Sprites](./sprites.md)
 - [Creating Sprites]()
-- [...]()
+- [Using Spritesheets]()
+- [SpriteFrame Cache]()
+- [Sprite Manipulation]()
+- [Polygon Sprite]()
 
 ## Actions
-
-- [By and To, what is the difference](./actions.md)
+- [What are Actions](./actions.md)
 - [Basic Actions]()
 - [Sequences]()
-- [...]()
+- [Sequence Internals, what is really happening]()
 
 ## Building and Transitioning Scenes
-
 - [What is a Scene](./scenes.md)
 - [Creating a Scene]()
-- [...]()
+- [Transitioning from Scene to Scene]()
 
 ## UI Components
-
- - [Widgets, oh, my!](./ui_components.md)
- - [Label]()
- - [Menu]()
- - [Buttons]()
- - [...]()
+- [Widgets, oh, my!](./ui_components.md)
+- [Label]()
+- [Menu]()
+- [Buttons]()
+- [Checkbox]()
+- [Loading Bar]()
+- [Slider]()
+- [TextField]()
 
 ## Other Node Types
-
 - [TileMap](./other_node_types.md)
 - [Particle System]()
 - [Parallax]()
 
 ## Event Dispatcher
-
 - [What is the EventDispatch mechanism](./event_dispatch.md)
 - [5 types of event listeners]()
 - [FixedPriority vs SceneGraphPriority]()
 - [Touch Events]()
-- [...]()
+- [Keyboard Events]()
+- [Accelerometor Events()
+- [Mouse Events]()
+- [Creating Custom Events]()
+- [Registering Events with the Dispatcher)[]
 
 ## 3D
-
 - [Terminology](./3d.md)
 - [Sprite3D]()
 - [Animation]()
-- [...]()
+- [Camera]()
+- [Cubemap Texture]()
+- [Skybox]()
+- [Lighting]()
+- [Terrain]()
+- [Tools for creating 3D Sprites]()
+- [Advanced 3D Topics]()
 
 ## Scripting
-
 - [Script component](./scripting.md)
+- [Using Scripts]()
 
 ## Physics
-
 - [Physics is scary, do I really need it? Please tell me no!](./physics.md)
 - [Physics terminology and concepts]()
-- [...]()
+- [Collision]()
+- [Queries]()
+- [Debugging/Disabling]()
 
 ## Audio
-
 - [Getting Started](./audio.md)
 - [Play background music]()
-- [...]()
-
-## File System Access
-
-- [Functions to read file]()
-
-## Networking with HTTP
-
-## Shaders and Materials
-
-- [What is a Shader]()
-- [Customizing Shaders]()
-- [...]()
-
-## How to optimize the graphics performance of your Cocos2d-x games
-
-- [Golden rules]()
-- [Simple checklist to make your Cocos2d-x game faster]()
+- [Play Sounds]()
+- [Audio operations]()
+- [SimpleAudio Engine vs Audio Engine]()
+
+## Advanced Topics
+- [File System Access]()
+- [Networking with HTTP]()
+- [Shaders and Materials]()
+- [Optimizing Graphics]()
+- SQLite
+
+## Installations
+- [Prerequisites]()
+- [Android - command-line]()
+- [Android - Visual Studio]()
+- [Android - Studio]()
+- [Android - Eclipse]()
+- [IOS]()
+- [Mac OS X]()
+- [Linux]()
+- [Tizen]()
+- [Windows]()
+- [Windows Phone]()
```

Comment:
Tizen and Windows Phone is not supported now, so i don't think they are needed.

Reply:
ok, removed